### DISCUSSION TITLE
cant install on Linux Ubuntu

### DISCUSSION THREAD

**Discussion by joelkahan** [Budibase/budibase#895] [Help] [answered]
i tried to run ./Budibase-Builder-0.3.8.AppImage
error: ./Budibase-Builder-0.3.8.AppImage: command not found

its can run on 64 bit?

**Comment by shogunpurple** [comment, marked as answer]
Hi @joelkahan, sorry to hear you are having trouble getting up and running with budibase.

I'm not able to recreate this issue on 64 bit linux - I followed the standard AppImage installation steps: 
- downloaded the AppImage 
- navigated to my downloads directory 
- updated the permissions (`chmod +x Budibase-Builder-0.3.8.AppImage`)
- ran the AppImage. (`./Budibase-Builder-0.3.8.AppImage`)

That's a really strange issue you are seeing. Generally when you run a dot slash command on linux and the executable isn't there, it will say `no such file or directory: ./Budibase-Builder-0.3.8.AppImage`. There could possibly a strange issue with your `PATH` variable on your machine causing this issue.

In any case, if you are on Ubuntu or any other debian based linux distribution, you should use the `.deb` installation of budibase. This will give you some nice benefits like auto update and full installation so you can use budibase like any other desktop application. You can download the 64 bit deb [here](https://github.com/Budibase/budibase/releases/download/v0.3.8/Budibase.Builder_0.3.8_amd64.deb). 

We will continue to debug and try to recreate the issue you saw, but please try the deb installation in the meantime and let us know if that works for you.

Thank you for using budibase and for reporting this to us!
